Multisig/UI/UI Library/RemoveSafeButton.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  RemoveSafeButton.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/13/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct RemoveSafeButton: View {
+    
+    @ObservedObject
+    var safe: Safe
+    
+    @State
+    var showDeleteConfirmation: Bool = false
+    
+    var body: some View {
+        Button(action: {
+            self.showDeleteConfirmation.toggle()
+        }) {
+            HStack {
+                Image("ico-remove")
```

Comment:
Have you tried the Image(systemName: "...").font(Font.body.bold()) to make system icon thick?